src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_sync/direct_query_sync_banner.tsx

```diff
@@ -109,8 +127,7 @@ export const DashboardDirectQuerySyncBanner: React.FC<DirectQuerySyncProps> = ({
                   )}`
                 : '--',
             },
-          })}
-
+          })}{' '}
```

Comment:
I think we don't need this `' '` since the scss should handle the margin here:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_sync/direct_query_sync_banner.scss